CHANGES.md

```diff
@@ -10,6 +10,7 @@
   * Added TTL 74381: arithmetic logic unit
   * Attribute sheet now honors application color theme.
   * Attribute sheet now displays HEX value of color properties.
+  * Fixed bugs in PortIO component and added the single bit version.
```

Comment:
Please be bit more verbose here. If we do not have any ticket ID to refer to, please at least add some words about what type of bugs user can consider fixed, because for now it is kinda unknown what really changed and where to look while using.

Reply:
Done